### PR TITLE
added start scripts to package.json

### PR DESCRIPTION
**Pull request by shaitaldesai**
Added all the start scripts to package.json.

### CODE REVIEW

**Comment by lee-philip-n** [comment]
You don't need to create a new branch for every new dependency. Just install it all in one branch then do one PR. I have no comments to change anything, but I can't merge due anything due to conflicts within your package.json. You may have to do some rebasing.

**Comment by shaitaldesai** [comment]
Can someone check my pr and merge? Thanks

**Comment by Angelhendrix** [comment]
I'll check it

File: package.json [hunk 1]
```diff
@@ -2,8 +2,10 @@
   "name": "cheerdango",
   "version": "1.0.0",
   "description": "",
-  "main": "webpack.config.js",
+  "main": "index.js",
```
Comment (Angelhendrix) [inline package.json]: looks good

**Pull request merged**